NPE with bulk pause resume

Probable something to do with aliases

```
[ERROR] 2025-10-02 08:41:43.035 [http-nio-17665-exec-10] BasicDispatcher - Cannot invoke "java.util.HashMap.containsKey(Object)" because the return value of "java.util.HashMap.get(Object)" is null
java.lang.NullPointerException: Cannot invoke "java.util.HashMap.containsKey(Object)" because the return value of "java.util.HashMap.get(Object)" is null
        at org.epics.archiverappliance.mgmt.bpl.BulkPauseResumeUtils.pauseResumePVs(BulkPauseResumeUtils.java:120) ~[classes/:?]
        at org.epics.archiverappliance.mgmt.bpl.ResumeArchivingPV.resumeMultiplePVs(ResumeArchivingPV.java:69) ~[classes/:?]
        at org.epics.archiverappliance.mgmt.bpl.ResumeArchivingPV.resumeMultiplePVs(ResumeArchivingPV.java:64) ~[classes/:?]
        at org.epics.archiverappliance.mgmt.bpl.ResumeArchivingPV.execute(ResumeArchivingPV.java:41) ~[classes/:?]
```

--- a/src/main/org/epics/archiverappliance/mgmt/bpl/BulkPauseResumeUtils.java
+++ b/src/main/org/epics/archiverappliance/mgmt/bpl/BulkPauseResumeUtils.java
@@ -96,9 +96,15 @@ public class BulkPauseResumeUtils {
         List<HashMap<String, String>> retVal = new LinkedList<HashMap<String, String>>();
         HashMap<String, HashMap<String, String>> retValMap = new HashMap<String, HashMap<String, String>>();
         LinkedList<String> realPVNames = new LinkedList<String>();
+        HashMap<String, String> incoming2real = new HashMap<String, String>();
         for (String pvName : pvNames) {
             String realName = configService.getRealNameForAlias(pvName);
-            if (realName != null) pvName = realName;
+            if (realName != null) {
+                incoming2real.put(pvName, realName);
+                pvName = realName;
+            } else {
+                incoming2real.put(pvName, pvName);
+            }
             realPVNames.add(pvName);
 
             HashMap<String, String> pvPauseResumeStatus = new HashMap<String, String>();
@@ -117,8 +123,9 @@ public class BulkPauseResumeUtils {
             }
         }
         for (String pvName : pvNames) {
-            if (retValMap.get(pvName).containsKey("validation")
-                    || retValMap.get(pvName).containsKey("status")) {
+            String realName = incoming2real.get(pvName);
+            if (retValMap.get(realName).containsKey("validation")
+                    || retValMap.get(realName).containsKey("status")) {
                 // We got some status
             } else {
                 retValMap

